web/src/components/Plugin/PluginPage.tsx

```diff
@@ -96,6 +98,14 @@ const PluginPage: React.FC<Props> = ({
       form.setFieldsValue({ plugin_config_id });
     });
   }, []);
+
+  useEffect(() => {
+    const openList = pluginList.filter(
+      (item) => initialData[item.name] && !initialData[item.name].disable,
+    );
+    setEnablePluginsList(openList);
+  }, [initialData]);
+
   const openPluginList = pluginList.filter(
```

Comment:
Can you help us improve this section, please? Like you've done here? 

https://github.com/Morakes/apisix-dashboard/blob/2877db87a5c3d8af1e187f012c406bd38f27f24a/web/src/components/Plugin/PluginPage.tsx#L102-L107